README.md

```diff
@@ -1,5 +1,5 @@
 # single-spa.js.org
-[![CircleCI](https://circleci.com/gh/CanopyTax/single-spa.js.org.svg?style=svg)](https://circleci.com/gh/CanopyTax/single-spa.js.org)
+[![CircleCI](https://circleci.com/gh/single-spa/single-spa.js.org.svg?style=svg)](https://circleci.com/gh/single-spa/single-spa.js.org)
```

Comment:
CircleCI already picked up on the repo rename and is working properly. See https://circleci.com/gh/single-spa/single-spa.js.org